### DISCUSSION TITLE
[Feature] Merge people (App)

### DISCUSSION THREAD

**Discussion by amgschnappi** [immich-app/immich#9197] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I would like to merge an unnamed face with Person1. For that, i am forced to use web interface (either laptop or my smartphone). Not possible with immich app. Please provide this functionality. Thanks. 

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#6259
